source/install/install-ubuntu-2004-postgresql.rst

```diff
@@ -47,7 +47,7 @@ Assume that the IP address of this server is 10.10.10.1.
 
   ``exit``
 
-9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` in a text editor as *root* user. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
+9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. You will need to replace `{version}` with the version PostgreSQL is currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. Replace `{version}` with the version of PostgreSQL that is currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Reply:
Thank you! We use the same hard-coded convention for the Ubuntu 18.04 installation documentation as well (https://docs.mattermost.com/install/install-ubuntu-1804.html#) Could you make the same change there for consistency, please?